metaflow/flowspec.py

```diff
@@ -499,6 +501,33 @@ def _validate_ubf_step(self, step_name):
             )
             raise InvalidNextException(msg)
 
+    def _get_foreach_item_value(self, item):
```

Comment:
nit: Add typing information here as well since you add it in the docs (I know we are not consistent but little by little :) )

Reply:
added